Description: Add Debian multiarch support
Author: Jean Baptiste Favre <debian@jbfavre.org>
Origin: other
Last-Update: 2017-03-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/dbms/CMakeLists.txt
+++ b/dbms/CMakeLists.txt
@@ -93,7 +93,7 @@ if (USE_STATIC_LIBRARIES)
 else ()
 	add_library(dbms SHARED ${dbms_headers} ${dbms_sources})
 	set_target_properties (dbms PROPERTIES SOVERSION ${VERSION_MAJOR} VERSION ${VERSION_SO} OUTPUT_NAME clickhouse)
-	install (TARGETS dbms LIBRARY DESTINATION lib COMPONENT clickhouse)
+	install (TARGETS dbms LIBRARY DESTINATION lib/${CMAKE_LIBRARY_ARCHITECTURE} COMPONENT clickhouse)
 endif ()
 
 
